pcsx2/IPU/IPUdma.cpp

```diff
@@ -101,8 +100,6 @@ void IPU1dma()
 		}
 		ipu1ch.madr = ptag[1]._u32;
 
-		ipu1cycles += 1; // Add 1 cycles from the QW read for the tag
```

Comment:
if instead of removing this you could change it to use totalqwc, that would be preferred (I thought I'd already converted this)